config/config.yaml

```diff
@@ -0,0 +1,73 @@
+samples: config/samples.tsv
+units: config/units.tsv
+
+# the sequencing adapter
+adapter: "ACGGATCGATCGATCGATCGAT"
+
+resources:
+  ref:
+    transcriptome: "resources/ref/Mus_musculus.GRCm38.cdna.all.plus_RUNX1a_GFP.fa"
+    # species needs to be an identifier known to biomart, e.g. mmusculus, hsapiens
+    species: mmusculus
+  ontology:
+    # gene ontology to download, used e.g. in goatools
+    gene_ontology: "http://current.geneontology.org/ontology/go-basic.obo"
+    # file name to use for downloaded gene ontology
+    gene_ontology_file: "resources/ontology/gene_ontology.obo"
+    # filename to save ensemble gene to GO term mapping to (downloaded via biomart)
+    ens_gene_to_go_file: "resources/ontology/ens_gene_to_go.tsv"
```

Comment:
Jep, that is definitely a good point. This is not really a configuration.  I think my original reasoning was having one instead of two points where you have to specify each file name -- but I guess that's over-engineering it. Changed it to hard-coded, now.

Reply:
https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/pull/4/commits/ce79cdee8f95cba4b75c276afeaf6010a4acbb2d